From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: vdso: add missing vdso_install target.
References: bnc#600516
Patch-mainline: Yes

Symptom:     Debugging vdso code is not possible.
Problem:     vdso debug symbols are not being installed.
Solution:    Add missing vdso_install target to install the unstripped vdso
             images into $(MODLIB)/vdso/.

Acked-by: John Jolly <jjolly@suse.de>
---
 arch/s390/Makefile |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/s390/Makefile
+++ b/arch/s390/Makefile
@@ -113,6 +113,12 @@ zfcpdump:
 archclean:
 	$(Q)$(MAKE) $(clean)=$(boot)
 
+vdso_install:
+ifeq ($(CONFIG_64BIT),y)
+	$(Q)$(MAKE) $(build)=arch/$(ARCH)/kernel/vdso64 $@
+endif
+	$(Q)$(MAKE) $(build)=arch/$(ARCH)/kernel/vdso32 $@
+
 # Don't use tabs in echo arguments
 define archhelp
   echo  '* image           - Kernel image for IPL ($(boot)/image)'
